src/application/App.tsx

```diff
@@ -36,97 +84,125 @@ const GET_OPERATION_COUNTS: TypedDocumentNode<
 `;
 
 export const App = () => {
+  const mountedRef = useRef(false);
   const [settingsOpen, setSettingsOpen] = useState(false);
   const { data } = useQuery(GET_OPERATION_COUNTS);
+  const [clientNotFoundModalOpen, setClientNotFoundModalOpen] = useState(false);
   const selected = useReactiveVar<Screens>(currentScreen);
-  const reloading = useReactiveVar<boolean>(reloadStatus);
+  const state = useReactiveVar(devtoolsState);
   const [embeddedExplorerIFrame, setEmbeddedExplorerIFrame] =
     useState<HTMLIFrameElement | null>(null);
 
-  // During a reload, reset the current screen to Queries.
   useEffect(() => {
-    if (reloading && selected !== Screens.Queries) {
-      currentScreen(Screens.Queries);
```

Comment:
I got rid of this weird reload behavior where it would always navigate you back to the queries screen anytime the page was reloaded. I couldn't think of a good reason why we couldn't just leave the user on the same screen they were looking at.